.gitlab-ci.yml

```diff
@@ -241,11 +241,21 @@ pkg:nix:
   stage: test
   variables:
     GIT_STRATEGY: none
+    # By default we use coq.cachix.org as an extra substituter but this can be overridden
+    EXTRA_SUBSTITUTERS: "${EXTRA_SUBSTITUTERS-https://coq.cachix.org}"
+    EXTRA_PUBLIC_KEYS: "${EXTRA_PUBLIC_KEYS-coq.cachix.org-1:Jgt0DwGAUo+wpxCM52k2V+E0hLoOzFPzvg94F65agtI=}"
```

Comment:
Just put in the value, project variables have precedence over job variables.

Reply:
Ah, great to know!